sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingSessionWindowSuite.scala

```diff
@@ -43,12 +43,17 @@ class StreamingSessionWindowSuite extends StreamTest
     val mergingSessionOptions = Seq(true, false).map { value =>
       (SQLConf.STREAMING_SESSION_WINDOW_MERGE_SESSIONS_IN_LOCAL_PARTITION.key, value)
     }
-    val providerOptions = Seq(
+    var providerOptions = Seq(
       classOf[HDFSBackedStateStoreProvider].getCanonicalName,
       classOf[RocksDBStateStoreProvider].getCanonicalName
     ).map { value =>
       (SQLConf.STATE_STORE_PROVIDER_CLASS.key, value.stripSuffix("$"))
     }
+    // RocksDB doesn't support Apple Silicon yet
```

Comment:
Maybe we should document this somewhere but could be done separately ..

Reply:
Yes, definitely. I'm currently working on this area and will do the similar with LevelDB.
I'll document it both together.